components/script/dom/testbinding.rs

```diff
@@ -493,6 +494,8 @@ impl TestBindingMethods for TestBinding {
     fn StringMozPreference(&self, pref_name: DOMString) -> DOMString {
         get_pref(pref_name.as_ref()).as_string().map(|s| DOMString::from(s)).unwrap_or_else(|| DOMString::new())
     }
+
+    fn DocumentOrTypedef(&self, _: Option<Option<DocumentOrBlobOrHTMLElementOrFormData >>) -> () {}
```

Comment:
Unfortunately this isn't what I'm looking for; the parameter should just be `Option<DocumentOrTypedefTest>`.